packages/core/src/interfaces/enums.ts

```diff
@@ -214,7 +214,7 @@ export enum AxisTitleOrientations {
 /**
  * enum of legend item type
  */
-export enum LegendItemType {
+export enum LegendItemTypes {
```

Comment:
Just want to make sure the `LegendItemType` -> `LegendItemTypes` change is something that we want, and isn't something that's going to trip up consumers? I might be missing something, but it just seems like an aesthetic change?